File: Python Word Examples/FieldsApiDemo/add_field/addMergeField.py
import spirecloudword
from spirecloudword import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey, baseUrl)
api = spirecloudword.api.fields_api.FieldsApi(configuration)

file_name = "Template.docx"
paragraph_path = "sections/0/paragraphs/0"
field_name = 'FiledTest'
extra = {'folder': 'input',
         'index_in_paragraph': 0,
         'dest_file_path': 'output/AddMergeField.docx'}
api.add_merge_field(file_name, paragraph_path, field_name, **extra)

File: JavaScript Word Examples/FieldsApiDemo/addMergeField.js
(function (){
    var appId = "your id";
    var appKey = "your key";
    var baseUrl = "https://api.e-iceblue.cn";

    var Spirecloudword = require('../../src/index');
    var apiClient = new Spirecloudword.ApiClient(appId, appKey,baseUrl);
    var instance = new Spirecloudword.FieldsApi(apiClient);
    
    var fileName = "AddField.docx";
    var paragraph_path = "Section/0/Body/0/Paragraph/0";
    var destFilePath = "output/Fields/AddMergeField_1.docx";
    var fieldName = 'FieldTest';
    var opts = { 'folder': 'input', 'indexInParagraph': 0 };
    instance.addMergeField(fileName, paragraph_path, fieldName, destFilePath, opts,
    function (error, data, response) {
         if (error) throw error;
    });
})();